lib/src/action.dart

```diff
@@ -65,6 +67,7 @@ class Action<T> implements Function {
   /// Cancel all subscriptions that exist on this [Action] as a result of
   /// [listen] being called. Useful when tearing down a flux cycle in some
   /// module or unit test.
+  @Deprecated('Use (and await) dispose() instead.')
```

Comment:
Yeah the deprecation resolutions are documented elsewhere in w_transport, but they should also be included in the doc comment for the deprecated member.

When I originally discovered the `Deprecated` annotation, I thought the message would show up with the deprecation hint that the analyzer gives you, but that's not the case. If all pertinent info can be included in a single line, then I think you should put it in the string description like so:

```dart
@Deprecated('Use (and await) dispose() instead. Will be removed in v3.');
```

If it requires a longer explanation on what to do instead or why it was deprecated, then I think you should just note when it will be removed and reference the doc comment:

```dart
/// Doc comment blah blah...
///
/// Deprecated: more information here about why this was deprecated as well
/// as a code example of what to do instead ..
@Deprecated('Will be removed in v3. See above for more information.')